### ISSUE TITLE
Helper puppeteer list throw error

### ISSUE THREAD

**Issue by othree** [Codeception/CodeceptJS#928]
#### What are you trying to achieve?

Use puppeteer helper. execute `codeceptjs list`

#### What do you get instead?

> Provide console output if related. Use `--verbose` mode for more details.

```bash
 Puppeteer I.grabAttributeFrom(async)
undefined:1
(this.this.options.fullPageScreenshots)
           ^

TypeError: Cannot read property 'options' of undefined
```

### Details

* CodeceptJS version: 1.14
* NodeJS Version: 9.4
* Operating System: macOS
* Puppeteer

This is an issue caused by arrizalamin/js-function-reflector#13 .

**Comment by reubenmiller** [comment]
Yeah I have already created a PR to fix this #923

**Comment by reubenmiller** [comment]
Duplicate #919
